src/lib.rs

```diff
@@ -101,7 +101,7 @@
 //!
 //! {{readme}}
 //!
-//! Some additional info here
+//! Some additional info here, like: "current version is {{version}}"
```

Comment:
Could you the `{{version}}` placeholder right below the `{{readme}}`? Like this:

```
//! {{readme}}
//!
//! Current version: {{version}}
```

And there is also the rendered example in the fenced block right below that.

Reply:
np :)